README.md

```diff
@@ -273,6 +273,10 @@ plugin to your Babel config:
 - `server` (default: `true`) - When `true` adds `serverSideRequirePath` config.
 - `webpack` (default: `false`) - When `true` adds `webpackRequireWeakId` config.
 
+**Note:**
+
+If you're using babel + webpack with [syntax-dynamic-import](https://babeljs.io/docs/plugins/syntax-dynamic-import/) plugin and it causes your server-side-rendering dont render on the first request. Consider using [babel-plugin-dynamic-import-node](https://github.com/airbnb/babel-plugin-dynamic-import-node) plugin for your server build. It'll convert **import()** to **require()** and ensure synchronous rendering.  
```

Comment:
> It'll convert **import()** to **require()** and ensure synchronous rendering. 

This isn't true, this is the basic transformation:

**Input:**
```js
import('test-module');
```

**Output:**
```js
Promise.resolve().then(() => require('test-module'));
```

Using this transform by itself will not ensure synchronous rendering. You'll still need to specify [`serverSideRequirePath`](https://github.com/thejameskyle/react-loadable#optsserversiderequirepath-optional) or use the [`react-loadable/babel` plugin](https://github.com/thejameskyle/react-loadable#babel-plugin).